waspc/data/Generator/templates/server/src/apis/types.ts

```diff
@@ -0,0 +1,20 @@
+{{={= =}=}}
+
+import prisma from '../dbClient.js'
+
+{=# isAuthEnabled =}
+import { {= userEntityUpper =} } from '../entities'
```

Comment:
Pro tip: use `import type`, see https://github.com/wasp-lang/wasp/pull/945#discussion_r1081399180.

Reply:
I see we're still importing several types without the `type` import (`Request` and `Response` from `express` for example).

This is not _that_ important, so feel free to skip going through them and fixing it.

I'm approving either way.

Reply:
Ah crap, I forgot to check others, I will update them before merging 👍🏻